contributing.md

```diff
@@ -47,6 +47,17 @@ and start hacking:
 $ git checkout -b my-feature-branch
 ```
 
+### GITHUB PACKAGES
+
+We use the Github Package registry to host some of our internal packages for tooling (e.g. linting, testing, etc.). In order to download packages from the Github registry, you need to generate an access token for Github and use it to login to the npm CLI. If you bootstrap the repository via Lerna (as described in the proceeding section), you should not receive any errors related to this, however, we still recommend that you generate an access token and use it with `npm login`:
+
+1. Go to https://github.com/settings/tokens
+2. Generate a new access token with the `repo` and `read:packages` scopes.
+3. Run `npm login --registry=https://npm.pkg.github.com`
```

Comment:
does it proxy through git to npm ?

Reply:
are these public pkg ?  If so why do you need to login ?

Reply:
The login command will just set the correct access token for the given registry. You still need to tell npm to use that registry as needed, which the `.npmrc` file in the repo will do for packages that are under the `@azuread` scope (i.e. setting an access token for a registry is different from using it).

And yes, I believe the Github registry will proxy to the npm registry for packages it doesn't have.

Reply:
Yes, these packages are public, but Github still requires that everyone login with their Github account (probably to help prevent DDOS and other abuse).